src/st/e_explosion_puff_opaque.h

```diff
@@ -36,12 +36,12 @@ void EntityWargExplosionPuffOpaque(Entity* self) {
     case 0:
         InitializeEntity(g_EInitParticle);
         params = self->params & 0xF;
-        obj = &D_8018129C[params];
-        self->palette = obj->palette + 0xD0;
+        obj = &D_80180FE0[params];
+        self->palette = obj->palette + E_PUFF_OPAQUE_PALETTE_OFFSET;
```

Comment:
Do we have an established pattern or preference for things like this? Currently I've just done stage level `#define`s inside the relevant headers